modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,68 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which can be configured by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
+
+The following annotations are supported by Knative Serving:
+
+* Disabling routes, which is specified by the `serving.knative.openshift.io/disableRoute` annotation.
+* Enabling pass-through, which is specified by the `serving.knative.openshift.io/enablePassthrough` annotation.
+
+.Prerequisites
+
+* You must have the {ServerlessOperatorName} and Knative Serving installed on your {product-title} cluster.
+
+.Procedure
+
+. Create a Knative service that contains the label or annotation that you want to propagate to the {product-title} route:
++
+.Example service YAML
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+  labels:
+    <label_name>: <label_value>
+  annotations:
+    <annotation_name>: <annotation_value>
+...
+----
++
+.Example of disabling routes by using the annotation
```

Comment:
The whole title for this section is: "Customizing labels and annotations for {product-title} routes"

Why we adding an example of "disabling routes by using the annotation"? Such example is already discussed in detail in include::modules/serverless-openshift-routes.adoc[leveloffset=+1].

I think what makes sense here is to showcase the passthrouth of those labels/annotations.

I've invoked the following command:

```bash
$ kn service create showcase \
  --image=quay.io/cardil/knative-serving-showcase \
  --annotation annot-a=b \
  --label label-c=d
```

And I see it did pass the label and annotation to a route:

```bash
$ kubectl get routes.route.openshift.io \
     -l serving.knative.openshift.io/ingressName=showcase \
     -l serving.knative.openshift.io/ingressNamespace=default \
     -n knative-serving-ingress -o yaml | egrep 'label-c|annot-a'
      annot-a: b
      label-c: d
            f:annot-a: {}
            f:label-c: {}
```

@nak3 @markusthoemmes Is that make sense to make this paragraph about passthrouth? If now, why the above section can't be used?

Reply:
I used this because it is a simple example - it doesn't matter if the example is shown elsewhere, or whether we show disable routes or passthrough, I was only including it to show how an annotation is added. I'll change it to passthrough.